docs/Installation-Linux.md

```diff
@@ -30,7 +28,7 @@ All the commands shown on the following instructions are meant to be run as the
 		sudo apt update
 
 		# Install dependencies
-		sudo apt install bash clang-6.0 coreutils fakeroot git grep make openssh-client perl rsync sed
+		sudo apt install bash coreutils fakeroot git grep make openssh-client perl rsync sed unzip libtinfo5
```

Comment:
If these are _additional_ dependencies, then we don't have to put unzip and libtinfo5 again

Reply:
I always read it as Linux/WSL and GCS having their own sets of dependencies. 

If the GSC step is meant to be additional dependencies, then git, fakeroot, and perl should be removed as well, as they are present in both Linux/WSL and GSC's dependency install commands.